files/image_config/caclmgrd/caclmgrd

```diff
@@ -182,6 +182,10 @@ class ControlPlaneAclManager(object):
                 acl_rules = {}
 
                 for ((rule_table_name, rule_id), rule_props) in self._rules_db_info.iteritems():
+                    if not rule_props:
+                        log_warning("rule_props for rule_id {} empty or null!".format(rule_id))
+                        continue
+
                     if rule_table_name == table_name:
                         acl_rules[rule_props["PRIORITY"]] = rule_props
```

Comment:
>rule_props["PRIORITY"] [](start = 34, length = 22)

If rule_props is not empty, but no PRIORITY, should we treat it as an error and keep the caclmgrd still running?

Reply:
Done: https://github.com/Azure/sonic-buildimage/pull/2475/commits/739939ad1caa733ce80c584a9f386d3cdc98674b